### ISSUE TITLE
Publish to GitHub Actions Marketplace

### ISSUE THREAD

**Issue by curtisma** [ChartBoost/ruff-action#12]
I was surprised that I couldn't find this action on [the GitHub Actions Marketplace.](https://github.com/marketplace?category=&type=actions&verification=&query=ruff)  Once published, it would be the ruff check with the highest number of stars.

You can just follow the GitHub Help instructions on [publishing actions to the marketplace](https://docs.github.com/en/actions/creating-actions/publishing-actions-in-github-marketplace).

**Comment by brucearctor** [comment]
Thanks for suggesting @curtisma -- let me dig into the process here.  

I haven't ever used GH Actions Marketplace, do you think it is commonly used [ i am guessing based on your question, it is your go-to for discovering actions ]?

**Comment by curtisma** [comment]
Yeah, I think it's pretty commonly used and can make it easier to find this action.  It also marks the repo as an action with a link to the action's page in the marketplace.

**Comment by DavidJFelix** [comment]
I think this would be a good issue to migrate over to the new location: https://github.com/astral-sh/ruff-action

**Comment by curtisma** [comment]
Done: https://github.com/astral-sh/ruff-action/issues/5

**Comment by charliermarsh** [comment]
Done: https://github.com/marketplace/actions/ruff-action
